Sources/Run/Services/ProductPaymentHandler.swift

```diff
@@ -98,17 +99,19 @@ extension Droplet {
             return Response(status: .noContent, body: "")
         }
         
-        guard result.status == .ok else {
+        if result.status != .ok {
```

Comment:
p2: why change  what's not broken? do you not like guard stylistically?

Reply:
so, what happened recently, when I opened the page, and did nothing for a few minutes I've received the "EAH! ..." It seems like the time out send the ok status as well. So I fixed that and was about to test but have got a merge conflict :) Which I'm trying to resolve on staging now.